odoo/addons/base/models/res_users.py

```diff
@@ -262,9 +262,18 @@ class Users(models.Model):
     _order = 'name, login'
 
     # User can write on a few of his own fields (but not his groups for example)
-    SELF_WRITEABLE_FIELDS = ['signature', 'action_id', 'company_id', 'email', 'name', 'image_1920', 'lang', 'tz']
+    @property
+    def SELF_WRITEABLE_FIELDS(self):
```

Comment:
maybe we should convert the above comment into a docstring + say it's meant to be overridden when adding fields to `res.users` that the user himself can read/write to?